Stellar-operation-manage-key-value.x

```diff
@@ -2,26 +2,30 @@
 
 namespace stellar
 {
-
+    //: Actions that could be performed on KeyValueEntry
     enum ManageKVAction
     {
         PUT = 1,
         REMOVE = 2
     };
 
+    //: `ManageKeyValueOp` is used to create manage key-value operation which on successful application will update the key-value entry presented in the system
```

Comment:
//: `ManageKeyValueOp` is used to create the manage key-value operation which, if performed successfully, will update the key-value entry presented in the system